tests/single.js

```diff
@@ -0,0 +1,70 @@
+const webdriver = require('selenium-webdriver');
+const { By } = require('selenium-webdriver');
+const assert = require('assert');
+// Input capabilities
+var { singleTestCapabilities } = require('../conf');
+
+async function runTestWithCaps (capabilities) {
+  let driver = new webdriver.Builder()
+    .usingServer(`https://hub-cloud.browserstack.com/wd/hub`)
```

Comment:
```suggestion
    .usingServer(`https://hub.browserstack.com/wd/hub`)
```

Reply:
Let's just move hubUrl `hub.browserstack.com/wd/hub` out to conf file and import in test files, would need to change at one palce if needed.